utils/build.config

```diff
@@ -11,7 +11,8 @@ OFI = v1.15.1
 MERCURY = v2.2.0
 PSM2 = PSM2_11.2.78
 PROTOBUFC = v1.3.3
-UCX=v1.12.1
+UCX=v1.13.0
 
 [patch_versions]
 spdk=https://github.com/spdk/spdk/commit/b0aba3fcd5aceceea530a702922153bc75664978.diff
+mercury=https://raw.githubusercontent.com/daos-stack/mercury/master/na_ucx_local_addr.patch
```

Comment:
Sorry, this is actually wrong.  You can't use master branch here but need to reference it by hash.

https://raw.githubusercontent.com/daos-stack/mercury/d993cda60d9346d1bd3451f334340d3b08e5aa42/na_ucx_changes.patch

Otherwise, it will break future builds if mercury repo removes the patch at some future date.  We've been through this pain many times.